src/style.css

```diff
@@ -1 +1,98 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url("fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 800;
+  font-style: normal;
+}
+
+:root {
+  --border: 1px solid #e1e7ed;
+  --box-shadow-input: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  --box-shadow-input-efect: 0 3px 8px 0 rgba(61, 78, 97, 0.20);
+}
+
+body {
+  margin: 0;
+}
+
+input {
+  font-family: Avenir-Book, sans-serif;
+  border: none;
+  font-weight: 300;
+}
+
+.form {
+  padding: 0 10px;
+}
+
+.form__form-big {
+  margin: 20px 0;
+}
+
+.form__form-big, .form__form-sm {
+  position: relative;
+}
+
+.form__form-big__input, .form__form-sm__input {
+  width: 100%;
+  border: var(--border);
+  box-shadow: var(--box-shadow-input);
+  border-radius: 4px;
+}
+
+.form__form-big__input {
+  padding: 23px 60px;
```

Comment:
This padding cut the image as you can see and that's one of the problems keeping your tests from being successful.